hyperstone/__init__.py

```diff
@@ -3,25 +3,21 @@
 from hyperstone.emulator import HyperEmu
 from hyperstone.engine import start, prepare
 from hyperstone.plugins import Plugin, RunnerPlugin
-from hyperstone.settings import Settings, SettingsType
+from hyperstone.settings import Settings
 from hyperstone.util import log, LazyResolver
-from hyperstone import hooks, plugins, emulator, engine, exceptions, settings
+from hyperstone import hooks, exceptions, plugins
 
 __all__ = [
     'megastone',
     'HyperEmu',
     'prepare',
     'start',
+    'plugins',
+    'hooks',
+    'exceptions',
     'Plugin',
     'RunnerPlugin',
     'Settings',
     'log',
     'LazyResolver',
-
-    'hooks',
-    'plugins',
-    'emulator',
-    'engine',
-    'exceptions',
-    'settings',
 ]
```

Comment:
Merge conflict, please rollback (Except for removing SettingsType, you can keep it)